week2/Week_2 (DraftForClass).ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>When writing lesson plans, try to set learning goals with actionable adjectives. 'Understand' tends to be abiguous and not easily measurable/testable. Here are some suggestions for yours:</p><ol><li>Describe the qualities of a vector (and how it differs from a scalar)</li><li>Breakdown velocity into x and y components (in 2-d space)</li><li>Demonstrate how vector components can be added and/or subtracted (e.g., head-wind against velocity)</li><li>Calculate the x and y components of velocity for a real-world scenario (sail boat on a lake)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-wildbillcolorado/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>